ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -269,3 +272,24 @@ def post_to_callback_url(self, callback_url: str, result_message: OcrdResultMess
         }
         response = requests.post(url=callback_url, headers=headers, json=json_data)
         self.log.info(f'Response from callback_url "{response}"')
+
+    def create_queue(self, connection_attempts=1, retry_delay=1):
+        """Create the queue for this worker
+
+        Originally only the processing-server created the queues for the workers according to the
+        configuration file. This is intended to make external deployment of workers possible.
+        """
+        if self.rmq_publisher is None:
+            attempts_left = connection_attempts if connection_attempts > 0 else 1
+            while attempts_left > 0:
+                try:
+                    self.connect_publisher()
+                    break
+                except AMQPConnectionError as e:
+                    if attempts_left <= 1:
+                        raise e
+                    attempts_left -= 1
+                    sleep(retry_delay)
```

Comment:
If this is a systematic problem, why not wait before the first connection?

Reply:
Already merged but anyway: This is user controlled waiting. User specifies if and if so how long he/she wants to wait. There are cases when the queue is already running and no waiting is necessary when starting a worker. I have the case that queue and worker are started at the same time, this takes about 2 Secs and waiting is necessary.

Reply:
Yeah, good point, I thought as much, so it was more of an understanding question than a change request. Thanks for the explanation!